CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Contributing guidelines
+
+## Principles and vision
+
+### [Roadmap to `v1.1.x`](./ROADMAP.md)
+
+### go version
+
+This repository supports _at least_ the 2 latest stable versions of the language.
```

Comment:
Are there other major golang projects which act as dependencies to the ecosystem at large which have this sort of stability guarantee? Or are they wider than this?

Of note, Golang 1.20 was only released a year ago, which seems like a comparatively small window.

Reply:
My suggestion is that we commit to 2 releases (i.e. run CI on them). that doesn't mean necessarily that we ENFORCE people to upgrade (go.mod require version may lag behind)